### PR TITLE
Changes to use log2timeline style guide and removed unused methods #176

### PR DESCRIPTION
**Pull request by joachimmetz**

### CODE REVIEW

File: dftimewolf/lib/state.py [hunk 1]
```diff
@@ -19,32 +19,37 @@ class DFTimewolfState(object):
...
 
   Attributes:
-    errors: [(str, bool)] The errors generated by a module. These should be
-        cleaned up after each module run using the CleanUp() method.
-    global_errors: [(str, bool)] the CleanUp() method moves non critical errors
-        to this attribute for later reporting.
-    input: list, The data that the current module will use as input.
-    output: list, The data that the current module generates.
-    store: dict, store of arbitrary data for modules.
+    config: TODO.
```
Comment (joachimmetz) [inline dftimewolf/lib/state.py]: @tomchop I could use your input here

File: dftimewolf/lib/utils.py [hunk 2]
```diff
@@ -51,43 +52,12 @@ def import_args_from_dict(value, args, config):
...
-        key: import_args_from_dict(val, args, config)
+        key: ImportArgsFromDict(val, args, config)
         for key, val in value.items()
     }
   elif isinstance(value, tuple):
-    return tuple(import_args_from_dict(val, args, config) for val in value)
-  return value
-
-
-def check_placeholders(value):
```
Comment (joachimmetz) [inline dftimewolf/lib/utils.py]: Moved this into tests `_CheckPlaceholders`

Comment (tomchop) [reply dftimewolf/lib/utils.py]: Ack

File: dftimewolf/lib/state.py (hunk 1 above)
Comment (tomchop) [reply dftimewolf/lib/state.py]: Filled-in the `TODO`s

**Pull request merged**